static/docs/command-reference/import.md

```diff
@@ -139,22 +152,25 @@ deps:
       rev_lock: 0547f5883fb18e523e35578e2f0d19648c8f2d5c
 ```
 
-If the Git revision moves, such as a branch, this doesn't have much of an effect
-on the import/update workflow. However, for static refs such as tags (unless
-manually updated), or for SHA commits, `dvc update` will not have any effect on
-the import. In this cases, in order to actually "update" an import, it's
-necessary to **re-import the data** instead, by using `dvc import` again without
-or with a different `--rev`. For example:
+If the
+[Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+moves (e.g. branches), you may use `dvc update` to bring the data up to date.
+However, for typically static references (e.g. tags), or for SHA commits, in
```

Comment:
(e.g. tags), -> , like tags,

Again, not insisting. It simply sounds more fluent to me this way.

Reply:
I like "(e.g. tags)" because we have "(e.g. a branch)" in the previous sentence. And commit hashes (SHA-1) are not [references](https://git-scm.com/book/en/v2/Git-Internals-Git-References) technically speaking. But thanks!